resources/js/components/utils/AccessBarrier.tsx

```diff
@@ -47,29 +47,17 @@ export const AccessBarrier: ParentComponent<AccessBarrierProps> = (props) => {
       Fallback: () => <p>Nie masz uprawnień do tego zasobu</p>,
       roles: [],
       Error: () => <Navigate href="/login" />,
-      Pending: () => (
-        <div class="h-screen flex justify-center items-center">
-          <MemoLoader size={300} />
-        </div>
-      ),
+      Pending: MemoLoader,
```

Comment:
Trochę szczegół, ale chyba tworzymy klika tych MemoLoader i to miga. Da się może jakoś w miarę łatwo połączyć całe bazowe ładowanie pod jednym loaderem?

Reply:
Robię to dokładnie w innym PR, tym o przyspieszeniu ładowania :)